packages/ketcher-macromolecules/src/components/monomerLibrary/RnaPresetGroup/RnaPresetGroup.tsx

```diff
@@ -108,8 +108,8 @@ export const RnaPresetGroup = ({ presets, duplicatePreset, editPreset }) => {
     [dispatch],
   );
 
-  const debouncedShowPreview = useMemo(
-    () => debounce((p) => dispatchShowPreview(p), 500),
+  const debouncedShowPreview = useCallback(
+    debounce((p) => dispatchShowPreview(p), 1000),
```

Comment:
There is a requirement for 2sec debounce. Should we change it?

Reply:
Changed for now, will sync with Ljubica as it seems a bit too long for me

Reply:
Reverted back to 500 as Ljubica noted it's not an expected change, she adjusted requirements as well